README.md

```diff
@@ -59,6 +59,8 @@ Using Gradle makes building WPILib very straightforward. It only has a few depen
 
 On macOS ARM, run `softwareupdate --install-rosetta`. This is necessary to be able to use the macOS x86 roboRIO toolchain on ARM.
 
+On linux, run `sudo apt install gfortran`. This is necessary to be able to build WPIcal on linux platforms.
```

Comment:
Is there a cmake check we can add for this?

Reply:
Build-only. libgfortran should get statically linked into the build. As for a CMake check, I think specifying `Fortran` as a language in the `project` call should do it (as well as `C` and `CXX`)

Reply:
Should this be done in the overall wpilib cmake or the wpical cmake?

Reply:
Just the wpical CMake file, because it applies only to wpical (the other projects already auto-detect the lanaguage.)